build.conf

```diff
@@ -39,29 +38,18 @@ ARMBIAN_IMG_7z=""
 ARMBIAN_IMG=""
 ARMBIAN_VERSION=""
 ARMBIAN_KERNEL_VERSION=""
-# Offset of the Armbian image for rootfs, in 512 bytes sectors
-ARMBIAN_IMG_OFFSET="8192"
+
+# Offset and sector size of the Armbian image for rootfs (found automatically if not set)
+IMG_OFFSET="" # 8192
+IMG_SECTOR="" # 512
 # this is a base name, we will work with partitions
 BASE_IMG=${TIMAGE_DIR}/base_image
 # how much will we increase the original image in MB
 BASE_IMG_ADDED_SPACE=768
-IMG_LOOP=""
-
-# Go vars
-# WARNING! this must match the ones defined in skywire
-# see http://github.com/SkycoinProject/skywire
-HOME=/root
-GOROOT=/usr/local/go
-GOPATH=/usr/local/skywire/go
-
-# Skywire vars
-SKYCOIN_DIR=${GOPATH}/src/github.com/SkycoinProject
-SKYWIRE_DIR=${SKYCOIN_DIR}/skywire
-SKYWIRE_GIT_URL="https://github.com/SkycoinProject/skywire.git"
 
-# Skywire IP settings
+# Skywire settings
 SKYMINER_GATEWAY="192.168.0.1"
-SKYMINER_MANAGER="192.168.0.2"
+SKYMINER_HYPERVISOR_PK=""
 SKYMINER_NODES="192.168.0.3 192.168.0.4 192.168.0.5 192.168.0.6 192.168.0.7 192.168.0.8 192.168.0.9"
```

Comment:
These might not be necessary anymore, since we do not have communication over local network between nodes anymore.